buildSrc/src/main/kotlin/Crates.kt

```diff
@@ -4,7 +4,7 @@
  */
 
 object Crates {
-    val AWS_RUNTIME = listOf(
+    val AWS_SDK_RUNTIME = listOf(
```

Comment:
just curious way all caps and underscores for these instead of `AwsSdkRuntime`?

Reply:
Just what I'm used to for constants from previous projects. No strong preference either way.